docs/reference/scripting.rst

```diff
@@ -690,6 +725,38 @@ Properties
     **currentMapView** : :ref:`script-mapview` |ro|, "Access the current map view."
     **tilesetsView** : :ref:`script-tilesetsview` |ro|, "Access the Tilesets view."
 
+.. _script-mapformatwrapper:
+
+Map Format Wrapper
+^^^^^^^^^^^^^^^^^^
+
+This is an object that can read or write map files. (Since 1.4)
+
+Properties
+~~~~~~~~~~
+
+.. csv-table::
+    :widths: 1, 2
+
+    **name** : string |ro|, The name of this format.
+    **extension** : string |ro|, "The file extension of this format, or `undefined`
+    if not available. Use `supportsFile` instead."
+
+Functions
+~~~~~~~~~
+
+MapFormatWrapper.read(fileName : string) : :ref:`script-map`
```

Comment:
I would not include the "Wrapper" in the name in the documentation, since I consider this just an implementation detail. Similar to how I don't include "Editable" in the documented type names.